src/cpu/flexcpu/simple_dataflow_thread.cc

```diff
@@ -223,7 +273,8 @@ SDCPUThread::onPCTranslated(Fault fault, const RequestPtr& req)
     if (!_cpuPtr->requestInstruction(req,
         [this](TheISA::MachInst i) {
             onInstructionFetched(i);
-        })) {
+        })
+    ) {
         panic("The CPU rejected my fetch request and I haven't been programmed"
```

Comment:
The above is really hard to read. It would be better to use a variable for the callback.

```
auto callback = [this](TheISA::MachInst i) { onInstructionFetched(i); };
if (!_cpuPtr->requestInstruction(req, callback)) {
    panic("...");
}
```

In fact, it may be a good idea to do this in many places. I would say it's a good idea if there's anywhere that you use the callback inside of another statement like this.

Reply:
Good idea, noted.